src/prefect/tasks/jupyter/jupyter.py

```diff
@@ -0,0 +1,66 @@
+import nbconvert
+import nbformat
+import papermill as pm
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class JupyterTask(Task):
+    """
+    Task for running Jupyter Notebooks.
+    In order to parametrize the notebook, you need to mark the parameters cell as described in
+        the papermill documentation: https://papermill.readthedocs.io/en/latest/usage-parameterize.html
+
+    Args:
+        - notebook_path (string, optional): path to fetch the notebook from.
+            Can be a cloud storage path.
+            Can also be provided post-initialization by calling this task instance
+        - notebook_params (dict, optional): dictionary of parameters to use for the notebook
+            Can also be provided at runtime
```

Comment:
I think the `notebook_` prefix here is redundant, it's already a notebook task. I'd suggest switching to `path` and `parameters` (not `params`) would be clearer.